manifests/shared/deployments/datadog.yml

```diff
@@ -0,0 +1,14 @@
+---
+releases:
```

Comment:
yes, but for bosh-init we need to provide the tarball. That's the only reason we do it that way. Otherwise for CF we do it in the same way as all other releases. Once we have automated release builds, we can re-factor how we provide releases.

Also, https://github.com/cloudfoundry/bosh/issues/1364 - meaning we have to do some workarounds to upload releases specified in runtime/addons.